cacheops/query.py

```diff
@@ -261,6 +265,8 @@ def iterator(self):
                 if cache_data is not None:
                     results = pickle.loads(cache_data)
                     for obj in results:
+                        # Notify about cache hit
+                        self._send_post_lookup_signal(hit_cache=True)
```

Comment:
Would be called for each item in a queryset. If fetch is empty, but still a cache hit, then it won't be called.